app/helpers/supplier_helper.rb

```diff
@@ -2,6 +2,6 @@
 
 module SupplierHelper
   def supplier_link(supplier)
-    supplier.is_a?(LeadProvider) ? nil : edit_admin_delivery_partner_path(supplier)
+    supplier.is_a?(LeadProvider) ? supplier.name : govuk_link_to(supplier.name, edit_admin_delivery_partner_path(supplier))
```

Comment:
Eh, not really a link anymore, is it? :/

Reply:
Not that I have a good name. `supplier_details_element`?

Reply:
I think this is just temporary. I would just keep the name